src/qt/paymentserver.cpp

```diff
@@ -211,13 +211,23 @@ void PaymentServer::ipcParseCommandLine(int argc, char* argv[])
             {
                 CBitcoinAddress address(r.address.toStdString());
 
-                if (address.IsValid(Params(CBaseChainParams::MAIN)))
```

Comment:
This is, I think, the long term solution for paymentserver and chainparams: https://github.com/jtimon/bitcoin/commit/ae575131833790e464cc9d7b17651a882dfae1c0